### ISSUE TITLE
Accessibility: MAS4.2.1: Inspect: Inappropriate control type for the combo box

### ISSUE THREAD

**Issue by ds-ms** [OfficeDev/office-ui-fabric-react#3222]
### Bug Report

- __Package version(s)__:  ^4.51.1
- __Browser and OS versions__: 
Test Environment:
OS: Windows 10 RS2
OS version: 15063.540
Browser: Edge browser.
Edge Version: Microsoft Edge 40.15063.0.0
Build number : Version 16.122.26818.0

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix?  No

Requested priority: Normal

Products/sites affected: *.visualstudio.com

#### Describe the issue:
Narrator reads as "Application Framework, collapsed, menu"
But the behavior is different in Chrome

#### Actual behavior:
Control type for "Application Framework" is menu.

#### Expected behavior:
Control type  should be "Combo box" for "Application Framework".

User Impact:
Visually impaired users, who rely on screen reader will get confused with the inappropriate control type.

**Comment by michaelangotti** [comment]
We are currently closing out all issues preceding Fabric React v5. If you are still seeing the same issue, please open a new issue.
